test/e2e/horizontal_pod_autoscaling.go

```diff
@@ -0,0 +1,89 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+)
+
+const (
+	sleep = 10 * time.Minute
+)
+
+var _ = Describe("Horizontal pod autoscaling", func() {
+	var rc *ConsumingRC
+	f := NewFramework("horizontal-pod-autoscaling")
+
+	BeforeEach(func() {
+	})
+
+	AfterEach(func() {
+	})
+	It("should scale from 1 pod to 3 pods (scale resource: CPU)", func() {
```

Comment:
see here for reference https://github.com/kubernetes/kubernetes/blob/master/test/e2e/autoscaling.go#L66